doc/manual/manual.tex

```diff
@@ -1572,6 +1572,145 @@ \subsubsection{Free surface stabilization}
 
 An example of a simple model which uses a free surface may be found in Section \ref{sec:cookbooks-freesurface}.
 
+\subsection{Calculations with melt transport}
+\label{sec:melt_transport}
+
+The original formulation of the equations in section \ref{sec:equations} describes the movement of solid mantle material. These computations also allow for taking into account how partially molten material changes the material properties and the energy balance through the release of latent heat. However, this will not consider melt extraction or any relative movement between melt and solid and there might be problems where the transport of melt is of interest. Thus, \aspect{} allows for solving additional equations describing the behavior of silicate melt percolating through and interacting with a viscously deforming host rock. This is implemented using the approach of \cite{KMK2013} and changes the Stokes system to:
```

Comment:
One typically capitalizes "Section" when refering to a particular section in a document. It's also common to make sure there's a no-break space between "Section" and the section number, so replace `section \ref{...}` by `Section~\ref{...}`.

Omit the colon at the end, as the formulas may be offset but are logically just an extension of the sentence that precedes it.